libs/parallelism/algorithms/include/hpx/parallel/datapar/loop.hpp

```diff
@@ -475,28 +475,25 @@ namespace hpx { namespace parallel { namespace util {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    namespace detail {
-
-        template <typename ExPolicy, typename Iter, typename F>
-        HPX_HOST_DEVICE HPX_FORCEINLINE constexpr typename std::enable_if<
-            hpx::is_vectorpack_execution_policy<ExPolicy>::value, Iter>::type
-        tag_dispatch(hpx::parallel::util::detail::loop_n_t<ExPolicy>, Iter it,
-            std::size_t count, F&& f)
-        {
-            return hpx::parallel::util::detail::datapar_loop_n<Iter>::call(
-                it, count, std::forward<F>(f));
-        }
+    template <typename ExPolicy, typename Iter, typename F>
+    HPX_HOST_DEVICE HPX_FORCEINLINE constexpr typename std::enable_if<
```

Comment:
`std::enable_if_t`? as you have been updating the others? (not a must)

Reply:
@aurianer I have not looked at this file too much, just did the edits to conform to the namespace changes of the main facility. I'll try to have a look at this file separately.